theano/gpuarray/tests/test_linalg.py

```diff
@@ -212,18 +212,21 @@ def test_gpu_matrix_inverse(self):
 
         fn = theano.function([A], gpu_matrix_inverse(A), mode=mode_with_gpu)
         N = 1000
-        A_val = rand(N, N).astype('float32')
+        test_rng = np.random.RandomState(seed=int(theano.config.unittests.rseed))
```

Comment:
The idea is good, but `config.unittests.rseed` can be set to the string `"random"`, so the right function to call is actually:
`test_rng = np.random.RandomState(seed=utt.fetch_seed())`.